generators/server/templates/build.gradle.ejs

```diff
@@ -532,7 +532,11 @@ dependencies {
     <%_ } _%>
     <%_ if (prodDatabaseType === 'mariadb') { _%>
     implementation "org.mariadb.jdbc:mariadb-java-client"
+        <%_ if (!reactive) { _%>
     liquibaseRuntime "org.mariadb.jdbc:mariadb-java-client"
+        <%_ } else { _%>
+        implementation "org.mariadb:r2dbc-mariadb:${r2dbc_mariadb_version}"
```

Comment:
R2DBC driver versions are managed by Spring Boot dependencies / R2DBC BOM
```suggestion
        implementation "org.mariadb:r2dbc-mariadb"
```

Reply:
Unfortunately that dependency is only added recently, so it does not contain yet: https://github.com/r2dbc/r2dbc-bom/commit/3ee8980068c69bfdd6ad14d7d4bdc55b8e6b4ccc